Dockerfile

```diff
@@ -1,21 +1,27 @@
 FROM node:16-alpine
 
-WORKDIR /apps/server
+# Set environment variables
+# Environment Variable to signal that we are running production
+ENV NODE_ENV=production
+# Ontime Data path
+ENV ONTIME_DATA=/external/
+
+WORKDIR /Resources/extraResources/
```

Comment:
A-ha, this is what you meant with /Resources/extraResources annoying dir.

I agree. This is something related to electron, and we shouldn't put our docker users through it if we can avoid it

Reply:
the docker users shouldn't even notice it beeing like that the files they need to acces can be put anywhere by the ONTIME_DATA variable

Reply:
sure, is the reason that it makes it easier for us to predict the directory across platforms? resources/extraResources is quite a mouthful 😅